scripts/deploy/iks/deploy-mm-serving.sh

```diff
@@ -79,6 +79,7 @@ mv kustomize /usr/local/bin/kustomize
 sed -i.bak 's/newTag:.*$/newTag: '"$GIT_COMMIT_SHORT"'/' config/manager/kustomization.yaml
 sed -i.bak 's/newName:.*$/newName: '"$DOCKERSANDBOX_NAMESPACE\/modelmesh-controller"'/' config/manager/kustomization.yaml
 rm config/manager/kustomization.yaml.bak
+echo -e '\n  disabled: true' >> config/runtimes/torchserve-0.x.yaml
```

Comment:
so, once we have the Torchserve FVTs running, we will have to remove this line again?

Reply:
With a new node added yesterday, FVT including torchserve completed after a couple of tries. I just remove this line.

Reply:
Great thank you Chin!